src/holodeck/agents.py

```diff
@@ -251,6 +253,18 @@ def __repr__(self):
 
 
 class UavAgent(HolodeckAgent):
+    __MAX_ROLL = 6.5080
```

Comment:
I would add comments to these indicating where you got them from (ie the file in `holodeck-engine`)

Reply:
Done